services/shuttle-axum/src/lib.rs

```diff
@@ -39,5 +40,16 @@ impl From<axum::Router> for AxumService {
         Self(router)
     }
 }
-/// The return type that should be returned from the [shuttle_runtime::main] function.
+
+/// Return type from the `[shuttle_runtime::main]` macro for a Axum-based service.
+///
+/// # Example
+///
+/// ```rust,no_run
+/// #[shuttle_runtime::main]
+/// async example_service() ->
+///     ShuttleAxum<impl FnOnce(&mut ServiceConfig) + Send + Clone + 'static> {
+///     todo!()
+/// }
```

Comment:
Hmm, I don't think this looks like in the public docs: https://docs.shuttle.rs/examples/axum. Can we change it to be similar?

Reply:
I will update them all with these.